### PR TITLE
change the webservice module but does'n finish.

### PR DESCRIPTION
**Pull request by sourirer**
change the webservice module to match new rule but still failed

### CODE REVIEW

File: dubbo-samples-webservice/dubbo-samples-webservice-consumer/src/main/resources/spring/dubbo-samples-webservice-consumer.xml [hunk 1]
```diff
@@ -21,9 +21,9 @@
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.3.xsd
        http://dubbo.apache.org/schema/dubbo http://dubbo.apache.org/schema/dubbo/dubbo.xsd">
 
-    <dubbo:application name="demo-consumer"/>
+    <dubbo:application name="dubbo-samples-webservice-consumer"/>
 
-    <dubbo:registry address="zookeeper://127.0.0.1:2181"/>
-
-    <dubbo:reference id="demoService" check="false" interface="org.apache.dubbo.samples.webservice.DemoService"/>
+    <dubbo:reference id="demoService" check="false" timeout="3000"
```
Comment (kylixs) [inline dubbo-samples-webservice/dubbo-samples-webservice-consumer/src/main/resources/spring/dubbo-samples-webservice-consumer.xml:26]: 这里好像不需要修改，原来是从registry中获取到调用地址，只需要修改注册中心地址
```
<dubbo:registry address="zookeeper://${zookeeper.address:127.0.0.1}:${zookeeper.port:2181}"/>
<dubbo:reference id="demoService" check="false" interface="org.apache.dubbo.samples.webservice.DemoService"/>
```

File: dubbo-samples-webservice/dubbo-samples-webservice-provider/pom.xml [hunk 2]
```diff
@@ -24,122 +24,58 @@
...
-            <artifactId>netty-all</artifactId>
-            <version>${netty-all.version}</version>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-samples-webservice-interface</artifactId>
+            <version>1.0-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.apache.dubbo</groupId>
-            <artifactId>dubbo-samples-webservice-interface</artifactId>
+            <artifactId>dubbo-samples-webservice-consumer</artifactId>
```
Comment (kylixs) [inline dubbo-samples-webservice/dubbo-samples-webservice-provider/pom.xml:78]: provider 不需要依赖 dubbo-samples-webservice-consumer

File: dubbo-samples-webservice/dubbo-samples-webservice-consumer/src/main/resources/spring/dubbo-samples-webservice-consumer.xml [hunk 3]
```diff
@@ -21,9 +21,9 @@
...
-    <dubbo:application name="demo-consumer"/>
-
-    <dubbo:registry address="zookeeper://127.0.0.1:2181"/>
-
-    <dubbo:reference id="demoService" check="false" interface="org.apache.dubbo.samples.webservice.DemoService"/>
+    <dubbo:application name="dubbo-samples-webservice-consumer"/>
+    <dubbo:registry address="zookeeper://${zookeeper.address:127.0.0.1}:${zookeeper.port:2181}"/>
+    <dubbo:reference id="demoService" check="false" timeout="3000"
+                     interface="org.apache.dubbo.samples.webservice.DemoService"/>
+<!--                     url="webservice://${dubbo.address:127.0.0.1}:8080/dubbo_samples_webservice_provider_war/services/org.apache.dubbo.samples.webservice.DemoService"/>-->
```
Comment (kylixs) [inline dubbo-samples-webservice/dubbo-samples-webservice-consumer/src/main/resources/spring/dubbo-samples-webservice-consumer.xml:28]: 请删除多余的注释

**Review by kylixs — APPROVED** [review]
Thanks for contribution!

**Review by AlbumenJ — APPROVED** [review]
lgtm

**Pull request merged**